### ISSUE TITLE
Svg Problem: A non-numeric value encountered in /project/vendor/phenx/php-svg-lib/src/Svg/Surface/SurfaceCpdf.php:380

### ISSUE THREAD

**Issue by pablopaul** [dompdf/dompdf#1437]
I am running dompdf over laravel-dompdf 0.8.0 on PHP 7.1 and get the following error:

```
[2017-04-26 15:23:30] debug.ERROR: ErrorException: A non-numeric value encountered in /project/vendor/phenx/php-svg-lib/src/Svg/Surface/SurfaceCpdf.php:380
Stack trace:
#0 /project/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php(36): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'A non-numeric v...', '/home/vagrant/C...', 380, Array)
#1 /project/vendor/phenx/php-svg-lib/src/Svg/Surface/SurfaceCpdf.php(380): Raven_Breadcrumbs_ErrorHandler->handleError(2, 'A non-numeric v...', '/home/vagrant/C...', 380, Array)
#2 /project/vendor/phenx/php-svg-lib/src/Svg/Document.php(207): Svg\Surface\SurfaceCpdf->setStyle(Object(Svg\DefaultStyle))
#3 /project/vendor/phenx/php-svg-lib/src/Svg/Tag/AbstractTag.php(62): Svg\Document->before(Array)
#4 /project/vendor/phenx/php-svg-lib/src/Svg/Document.php(352): Svg\Tag\AbstractTag->handle(Array)
#5 [internal function]: Svg\Document->_tagStart(Resource id #78, 'svg', Array)
#6 /project/vendor/phenx/php-svg-lib/src/Svg/Document.php(223): xml_parse(Resource id #78, '<?xml version="...', false)
#7 /project/vendor/dompdf/dompdf/lib/Cpdf.php(4975): Svg\Document->render(Object(Svg\Surface\SurfaceCpdf))
#8 /project/vendor/dompdf/dompdf/src/Adapter/CPDF.php(834): Cpdf->addSvgFromFile('/home/vagrant/C...', 262.59, 420.75, 318.75, 318.75)
#9 /project/vendor/dompdf/dompdf/src/Renderer/Image.php(110): Dompdf\Adapter\CPDF->image('/home/vagrant/C...', 262.59, 52.5, 318.75, 318.75, 'normal')
#10 /project/vendor/dompdf/dompdf/src/Renderer.php(293): Dompdf\Renderer\Image->render(Object(Dompdf\FrameDecorator\Image))
#11 /project/vendor/dompdf/dompdf/src/Renderer.php(127): Dompdf\Renderer->_render_frame('image', Object(Dompdf\FrameDecorator\Image))
#12 /project/vendor/dompdf/dompdf/src/Renderer.php(194): Dompdf\Renderer->render(Object(Dompdf\FrameDecorator\Image))
#13 /project/vendor/dompdf/dompdf/src/Renderer.php(194): Dompdf\Renderer->render(Object(Dompdf\FrameDecorator\Block))
#14 /project/vendor/dompdf/dompdf/src/Renderer.php(194): Dompdf\Renderer->render(Object(Dompdf\FrameDecorator\Block))
#15 /project/vendor/dompdf/dompdf/src/FrameReflower/Page.php(148): Dompdf\Renderer->render(Object(Dompdf\FrameDecorator\Block))
#16 /project/vendor/dompdf/dompdf/src/FrameDecorator/AbstractFrameDecorator.php(893): Dompdf\FrameReflower\Page->reflow(NULL)
#17 /project/vendor/dompdf/dompdf/src/Dompdf.php(831): Dompdf\FrameDecorator\AbstractFrameDecorator->reflow()
#18 /project/vendor/barryvdh/laravel-dompdf/src/PDF.php(200): Dompdf\Dompdf->render()
#19 /project/vendor/barryvdh/laravel-dompdf/src/PDF.php(146): Barryvdh\DomPDF\PDF->render()
#20 /project/vendor/barryvdh/laravel-dompdf/src/PDF.php(158): Barryvdh\DomPDF\PDF->output()
```

My Template looks like this
```
<!doctype html>
<html lang="de">
    <head>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
        <title>project</title>
        <style type="text/css">

            @font-face {
                font-family: "Caveat";
                font-weight: normal;
                font-style: normal;
                font-variant: normal;
                src: url({{ storage_path('fonts/Caveat.ttf') }}) format("truetype");
            }

            @page { margin: 55px 50px; }

            body {
                font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
            }

            .header {
                position: fixed;
                left: 0px;
                top: -55px;
                right: -29px;
                height: 30px;
                padding: 5px;
                background-color: #f1f2ec;
                border-radius: 7px;
            }

            .header img {
                max-height: 100%;
            }

            img {
                max-width: 100%;
            }

            .row {
                width: 100%;
                margin: 0;
                padding: 0;
            }

            p {
                margin: 0;
                padding: 0;
            }

            .left {
                float: left;
                width: 33%;
                margin: 0;
                margin-right: -39%;
                page-break-inside: avoid;
                font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
            }

            .right {
                float: right;
                width: 61%;
                margin: 0;
                margin-left: 39%;
                page-break-inside: avoid;
                font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
            }

            .column {
                margin-bottom: 15px;
                padding: 15px;
                background-color: #f1f2ec;
                border-radius: 7px;
            }

            .Text--user {
                font-family: "Caveat";
                font-size: 24px;
            }

            .center {
                text-align: center;
            }

            .debug {
                border: 0px dotted grey;
            }
        </style>
    </head>

    <body>
        <div class="header">
            <img style="float: left; height: 100%;" src=".{{ $tenantLogo }}" />
            <img style="float: right; height: 100%;" src=".{{ '/images/header.png' }}" />
        </div>

        <div class="row">
            <div class="debug left column">
                {!! $tenantTexts['t125intro'] !!}
            </div>

            <div class="debug right column center">
                <img src=".{{ $areaImage }}" alt="">
            </div>
        </div>
        <div style="clear:both;"></div>

        <div class="row">
            <div class="left column debug">
                {!! $tenantTexts['t125resources'] !!}
            </div>

            <div class="right column center debug">
                <img src=".{{ object_get($userProcess, "voice_figure_set_4_stage_image_path") }}" alt="">

                <span class="Text--user">{{ object_get($userProcess, "voice_figure_set_2_voice_name") }} - </span>
                <span class="Text--user">{{ object_get($userProcess, "voice_figure_set_3_voice_name") }} - </span>
                <span class="Text--user">{{ object_get($userProcess, "voice_figure_set_4_voice_name") }}</span>
            </div>
        </div>
        <div style="clear:both;"></div>

        <div class="row">
            <div class="left column debug">
                <p>{!! $tenantTexts['t000summaryMatterCompressionBefore'] !!}</p>
            </div>
            <div class="right column debug">
                <p><i>{!! object_get($userProcess, "matter_compression_before") !!}</i></p>
            </div>
        </div>
        <div style="clear:both;"></div>

        <div class="row">
            <div class="left column debug">
                {!! $tenantTexts['t125matterCompressionAfter'] !!}
            </div>
            <div class="right column debug">
                <i>{!! object_get($userProcess, "matter_compression_after") !!}</i>
            </div>
        </div>
        <div style="clear:both;"></div>

        <div class="row">
            <div class="left column debug">
                <p>{!! $tenantTexts['t125conclusion1'] !!}</p>
            </div>
            <div class="right column debug">
                {!! $tenantTexts['t125conclusion2'] !!}
            </div>
        </div>
        <div style="clear:both;"></div>

        <div class="row debug">
            <strong>{{ $tenantTexts['t000summaryFooterCredits'] }} {{ $tenantTexts['tenantName'] }} ({{ Carbon\Carbon::today()->format('d.m.Y') }})</strong>
        </div>
        <div style="clear:both;"></div>

    </body>
</html>
```

`<img src=".{{ $areaImage }}" alt="">` **outputs an Svg file if I remove this the Pdf gets generated without error.**

The Svg can be found here: https://www.dropbox.com/s/t6g2vwfpm4pmfmw/sample.svg?dl=0

**Comment by hakimio** [comment]
As it's clearly svg rendering bug, you should close the bug report here and open a new one under appropriate project: https://github.com/PhenX/php-svg-lib

**Comment by hakimio** [comment]
Also, while you are at it, you should test the latest development version of "php-svg-lib". It seems recently there were some bug fixes, which should address the issue you are seeing here.

**Comment by PhenX** [comment]
Hello I just published a new release (v0.3) for php-svg-lib

**Comment by bsweeney** [comment]
see #1472
